pkg/objstore/s3/s3.go

```diff
@@ -196,6 +196,10 @@ func (b *Bucket) Iter(ctx context.Context, dir string, f func(string) error) err
 	}
 
 	for object := range b.client.ListObjects(b.bucket, dir, false, ctx.Done()) {
+		// catch the error when failed to list objects
```

Comment:
Can you make those comment full sentence (and below?) - upper case first letter and trailing period.

Reply:
Done